configs/components/oifs/oifs43.cmip.yaml

```diff
@@ -0,0 +1,24 @@
+oifs:
```

Comment:
For every new file I will be passing now a yaml formater that indents to 4. I will leave non-new files untouch in this PR so that we can see the diffs clearly.

In the future the yaml formater will run in the CI and make sure that everything is formatted correctly.

Reply:
great stuff!